test/test_utils/ecr.py

```diff
@@ -268,3 +269,26 @@ def populate_ecr_scan_with_web_scraper_results(
                 {"_comment": "Could Not be Processed. Webpage for this might not be in the required format."}
             ]
     return ecr_scan_to_be_populated
+
+
+def get_all_the_tags_of_an_image_from_ecr(ecr_client, image_uri):
+    """
+    Uses ecr descrive to ger all the tags of an image.
+    :param ecr_client: boto3 Client for ECR
+    :param image_uri: str Image URI
+    :return: list, All the image tags
+    """
+    account_id = image_uri.split('.')[0]
+    image_repo_uri, image_tag = image_uri.split(":")
+    _, image_repo_name = image_repo_uri.split("/")
+    response = ecr_client.describe_images(
+        registryId=account_id,
+        repositoryName=image_repo_name,
+        imageIds=[
+            {
+                'imageDigest': get_image_digest(image_uri),
```

Comment:
1. The `imageDigest` input is unnecessary if `imageTag` is supplied as a part of the `describe_images` call.
2. The `get_image_digest` function makes the assumption that the image has been pulled, when that doesn't necessarily have to be the case. We should not require a time-expensive operation such as pulling an entire image just to query the tags on the image on ECR.